src/components/ec-radio-btn-group/ec-radio-btn-group.spec.ts

```diff
@@ -0,0 +1,191 @@
+import type { MountingOptions } from '@vue/test-utils';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { mount } from '@vue/test-utils';
+
+import type { CVueWrapper } from '../../../tests/utils/global';
+import { EcRadioBtnGroup } from '../../main';
+import type { RadioButtonGroupProps } from './types';
+
+function mountRadioBtnGroup(props?: Partial<RadioButtonGroupProps>, mountOpts?: MountingOptions<RadioButtonGroupProps>): CVueWrapper {
+  return mount(
+    EcRadioBtnGroup as any, // eslint-disable-line
+    {
+      props: {
+        options: [
+          { value: 'y', label: 'Yes' },
+          { value: 'n', label: 'No' },
+        ],
+        name: 'testingNameProp',
+        ...props,
+      },
+      ...mountOpts,
+    },
+  ) as CVueWrapper;
+}
+
+describe('EcRadioBtn', () => {
+  it('should render properly', () => {
+    const wrapper = mountRadioBtnGroup();
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  describe('props', () => {
+    it('should render the child component (radio-btn) with all the given props', () => {
```

Comment:
In this test we are testing a Vue functionality about passing props and setting it by Vue for the component where we passed those. This tests not our code but Vue core, that we consider as tested properly by Vue core team.

If you want to test exactly what is written in `it`, then you should get all child components with `findAllComponents(EcRadioBtn)` and check props are passed in there.